src/snapshot/src/persist.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+//! Defines an abstract interface for saving/restoring a component from state.
+#![deny(missing_docs)]
```

Comment:
We should either add this to the `lib.rs` level, or remove the exclamation mark. The exclamation mark says that this macro applies globally, at the crate level. You typically add this in `src/lib.rs`. If we want to only deny the missing docs in this file, the exclamation mark is not needed.

Reply:
It is present in lib.rs. I'll remove it from here.